data/coalition/coalition missions.txt

```diff
@@ -1866,3 +1866,469 @@ mission "Saryd University Lecture"
 			`	"Thank you, captain! Expanded our knowledge greatly, you have. Hope to see you again, I do. Safe travels!"`
 			`	After he shakes your hand, you say your goodbyes and board your ship.`
 				decline
+
+
+
+mission "Coalition: Expeditions Past 1"
+	minor
+	name "Scouting Permit"
+	description "Transport Sedlitaris, a Heliarch Tribune, to the <destination>, where she will try to petition for the Heliarch council to allow her to go on a scouting expedition out of the Coalition. Payment is <payment>."
+	passengers 1
+	to offer
+		"coalition jobs" >= 40
+		random < 5
+	source
+		government "Coalition"
+	destination "Ring of Friendship"
+	on offer
+		require "Jump Drive"
+		conversation
+			`As you exit your ship and start heading to the spaceport, you hear a rushed galloping getting closer, and turn to see a Saryd Heliarch coming to you, gesturing for you to wait. She slows down once she sees you're obliging, and slowly walks to you, while glancing at your ship.`
+			`	"Apologize for cutting you off, I must, Captain <last>, but in need of extended transport services, I am," she says. "Sedlitaris Quatcid, my name is, a Heliarch tribune. Brought to <destination>, I must be. A petition to the council, I am to make."`
+			choice
+				`	"A 'petition?' What do you mean?"`
+				`	"I don't think you'd need some 'extended' service just for that."`
+				`	"Sorry, I'm not taking any more passengers right now."`
+					decline
+			`	"Nearly a thousand years ago, leading an expeditionary fleet to outside of the Coalition, an ancestor of mine was." She explains. "Mimodum, his name was, and after having risen the ranks and lifted our family from poorer conditions, chosen to lead the expedition he was. Only, return, he did not. To find out what happened, I wish. Looking for you, I have, as after brought to <planet>, I have been, and approved, my request is, need a transport to bring me to planets he visited, I will."`
+			choice
+				`	"I'd be glad to help you find out what happened. Let me show you to your bunk."`
+					goto accept
+				`	"Couldn't you just have gone on your own, or paid someone to bring you there if you don't have a ship?"`
+				`	"Actually I'm not going to be leaving Coalition space anytime soon, sorry."`
+					decline
+			`	She shakes her head. "Forbidden from leaving, anyone is. Steal a jump drive for that, I'd also need to do, and obviously illegal, that is. Allowed to head out, only Heliarchs who have received permission from the council can. Finally time to hear my request again, they have, so go to <planet>, I must."`
```

Comment:
The Heliarchs have stopped sending ships out because as the missions reveal all the latest expeditions never returned, so it's a waste of Jump Drives to be sending them out, as they see it. They only stick to the occasional visits to Poisonwood, which is just one jump away and relatively safe.
Sedlitaris in particular has been trying to get this permission for a while, and due to her insistence they revoked her rights to a Heliarch ship, just in case she tried something like taking a Jump Drive by force with said ship. She's hiring you because she knows the Heliarchs won't provide her a Jump Drive, much less a fleet.

I thought I had pointed to that enough, I don't know where exactly you'd like me to explain it more? Just comment if you have a specific part of the mission in mind for that.

Reply:
I'm sorry, I made this comment before I read the entire mission. It makes sense in retrospect. Feel free to resolve.